integration-tests/docker/docker-compose.yml

```diff
@@ -36,6 +36,7 @@ services:
     environment:
       - DRUID_INTEGRATION_TEST_GROUP=${DRUID_INTEGRATION_TEST_GROUP}
     depends_on:
+      - druid-overlord
```

Comment:
why this change?

Reply:
So, I copied this change over from the previous `docker-compose.override-config.yml` file. I'm not sure how it worked before, but now it fails without this change due to circular dependencies in this docker compose file.